core/src/com/unciv/ui/MultiplayerScreen.kt

```diff
@@ -223,71 +199,126 @@ class MultiplayerScreen() : PickerScreen() {
 
                 gameTable.add(gameButton).pad(5f).row()
                 leftSubTable.add(gameTable).row()
-            }catch (ex: Exception) {
-                //skipping one save is not fatal
-                ResponsePopup("Could not refresh!".tr(), this)
-                continue
             }
+            leftSideTable.clear()
+            leftSideTable.add(leftSubTable)
+        }
+    }
+
+    /* Is this worth the translation effort? Seems definitely more polished if used
```

Comment:
What do you think?
I don't really like to add to many new translation strings 😬